tracim/tracim/lib/content.py

```diff
@@ -134,7 +156,7 @@ def build_breadcrumb(self, workspace, item_id=None, skip_root=False) -> [Breadcr
         return breadcrumb
 
     def __real_base_query(self, workspace: Workspace=None):
-        result = DBSession.query(Content)
+        result = self.get_base_query()
```

Comment:
When you have 2 methods __real_base_query() and get_base_query(), we expect that  get_base_query() make a call to __real_base_query(). Here, it's the inverse, which is not intuitive.

Please rename methods in order to make the naming intuitive